### ISSUE TITLE
trainning settings

### ISSUE THREAD

**Issue by wtdfop** [ADiko1997/ReViT#2]
Could you offer your trainnig settings (including data augumentation methods, hyper parameters like learning rate, weight decay) while trainnig ReVit on small image sets like cifar10 and cifar100 from scratch? That is very important for me. Thank you
